From b9da3b2f4d45d3988ed07f9d313bd6a608c88981 Mon Sep 17 00:00:00 2001
From: LemonPie <lemonpie@artixlinux.org>
Date: Fri, 1 Dec 2023 19:19:51 +0000
Subject: [PATCH] disable pkgver() function

---
 PKGBUILD | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/PKGBUILD b/PKGBUILD
index 9eacd3f..35a889a 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -96,10 +96,10 @@ prepare() {
   git -c protocol.file.allow=always submodule update cmake/sanitizers-cmake
 }
 
-pkgver() {
-  cd dolphin-emu
-  git describe | sed 's/-/.r/; s/-g/./'
-}
+# pkgver() {
+#   cd dolphin-emu
+#   git describe | sed 's/-/.r/; s/-g/./'
+# }
 
 build() {
   export CFLAGS+=' -I/usr/include/mbedtls2'
-- 
2.43.0

